version_base.bzl

```
@@ -0,0 +1,21 @@
+# Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+# Exceptions. See /LICENSE for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+"""The base version of the Carbon repository.
+
+The Carbon SemVer version is defined here and then exported to various layers
```

Suggesting a small rephrasing here... For context, I'm trying to work through (a) trying to figure out how Carbon SemIR and Carbon SemVer relate (suggesting link below), and (b) "here and then" mentally parsing similar to "here and now".

Note, SemVer is not an abbreviation I was familiar with (I did a quick search to see if this was some common thing, but I could see others assuming Carbon SemVer is Carbon-specific). If you want to note it explicitly here, I'd suggest something like "This uses Semantic Versioning." as a direct statement.

```suggestion
Carbon's version is defined here for export to various layers
```